releasenotes/v1.10.0-beta.0.md

```diff
@@ -0,0 +1,166 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.9.0
+
+## :warning: Breaking Changes
+
+- Bump CAPI to v1.10.0-beta.0 (#2432)
```

Comment:
CAPI uplift is not breaking its Controller Runtime. PR was not titled properly, let's change it
```suggestion
- Bump CAPI to v1.10.0-beta.0 and Controller Runtime to 0.20.3 (#2432)
```

Reply:
Moved it to superseeded as suggested below.